### ISSUE TITLE
Mod manager Update

### ISSUE THREAD

**Issue by ChillyCoder792** [Umbranoxio/BeatSaberModInstaller#102]
needs and update for the Newest 0.13.0 version

**Comment by phiresky** [comment]
duplicate of #101. you should learn how github works before spamming people with emails.

**Comment by notyoyoma** [comment]
Suggestion: instead of "learn how github works", it would more polite to say "please search the existing issues before submitting new ones".

To be fair, any internet forum has similar etiquette, and you would hope people would translate general etiquette over to github.

**Comment by notyoyoma** [comment]
Hey @ChillyCoder792, If you're still dealing with this issue, @megalon explained the current state of the project [here](https://github.com/Umbranoxio/BeatSaberModInstaller/issues/103#issuecomment-474804108).

TLDR: [This new repo](https://github.com/lolPants/modsaber-installer/) should work, and get you playing again
